config/webpack/resolve.js

```diff
@@ -1,12 +1,12 @@
 /**
  * @author Adam Meadows [@job13er](https://github.com/job13er)
- * @copyright 2015 Cyan, Inc. All rights reserved.
+ * @copyright 2015 Ciena Corporation. All rights reserved.
  */
 
 'use strict';
 
 module.exports = {
     alias: {},
-    extensions: ['', '.webpack.js', '.web.js', '.js', '.6.js'],
+    extensions: ['', '.webpack.js', '.web.js', '.js', '.6.js', '.tsx', '.ts'],
```

Comment:
@danjres This and the above, plus the changes in `package.json` are pretty much all I did for TypeScript support so far.